migrate/20250110_add_kubernetes_models.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:kubernetes_cluster) do
+      column :id, :uuid, primary_key: true
+      column :name, :text, null: false
+      column :cp_node_count, :integer, null: false
+      column :kubernetes_version, :text, null: false
+      column :location, :text, collate: '"C"', null: false
+      column :created_at, :timestamp, null: false, default: Sequel::CURRENT_TIMESTAMP
+
+      check { (cp_node_count =~ [1, 3]) }
```

Comment:
```suggestion
      constraint(:check_cp_node_count, cp_node_count: [1, 3])
```

First, it's best to always give a constraint a name so it can be deterministically dropped later.  Second, I don't think the virtual row block is needed here, so switch to the simpler hash format.

Reply:
Actually, let me remove this constraint from the DB. IMO, it's a best practice to have odd number of CP nodes and having a DB constraint for that feels a bit too much.